src/please.go

```diff
@@ -566,10 +566,11 @@ var buildFunctions = map[string]func() int{
 		})
 	},
 	"somepath": func() int {
-		return runQuery(true,
-			[]core.BuildLabel{opts.Query.SomePath.Args.Target1, opts.Query.SomePath.Args.Target2},
+		a := utils.ReadStdinLabels([]core.BuildLabel{opts.Query.SomePath.Args.Target1})
+		b := utils.ReadStdinLabels([]core.BuildLabel{opts.Query.SomePath.Args.Target2})
```

Comment:
nit: `from` and `to` would make this code more "self documenting" ;P

Reply:
They don't actually have directionality though - you can pass the targets in either order